packages/plugin-senpi-rewards/src/utils/balance.ts

```diff
@@ -0,0 +1,33 @@
+import { createPublicClient } from "viem";
+import { http } from "viem";
+import { baseSepolia } from "viem/chains";
+
+export const getRewardBalance = async (
```

Comment:
Add retries here to call again if it failed to connect to the RPC and fetch data